x-pack/plugins/observability/public/pages/alerts/alerts_disclaimer.tsx

```diff
@@ -5,33 +5,61 @@
  * 2.0.
  */
 
-import React from 'react';
-import { EuiLink, EuiCallOut } from '@elastic/eui';
+import React, { useState } from 'react';
+import { EuiLink, EuiCallOut, EuiButton, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 
+const LOCAL_STORAGE_KEY_MSG_ACK = 'xpack.observability.alert.ack.experimental.message';
+
 export function AlertsDisclaimer() {
+  const getCurrentExperimentalMsgAckState = () => {
+    const isExperimentalMsgAck = localStorage.getItem(LOCAL_STORAGE_KEY_MSG_ACK);
```

Comment:
Very trivial but, there is a nice Storage implementation in `kibana_utils`. It Stringify on set and Parse on get.
So you can get rid of Boolean vs. String confusion and simply use `Storage.get(key)`.
IDK if such a cross plugin dependency is ok though. If not, we can copy it.
Just a suggestion, ultimately it's your decision :)

https://github.com/elastic/kibana/blob/main/src/plugins/kibana_utils/public/storage/storage.ts

Reply:
We can't import this module into the plugins folder. Only top-level public and server modules are allowed. 

However, I used `JSON.parse`. I prefer more code (using JSON.parse could throw an error, so try/catch added) instead of potential confusion.  👍🏻